stack.yaml

```diff
@@ -1,11 +1,11 @@
-resolver: nightly-2018-12-15
+resolver: lts-12.3
 
 packages:
 - .
 
 extra-deps:
-#- accelerate-1.2.0.1
-- bed-and-breakfast-0.5
+- primitive-0.6.1.1
 
 - git: https://github.com/adscib/monad-bayes
   commit: 647ba7cb5a98ae028600f3d828828616891b40fb
+- /root/repa-linear-algebra
```

Comment:
I would like to add `repa-linear-algebra` dependency via git link to your fork

Reply:
BTW, add please our test file that we've created yesterday.

Reply:
ok. I'll push the changes and replace that )